spec/support/factory_girl.rb

```diff
@@ -8,6 +8,10 @@
     password_confirmation 'password'
   end
 
+  factory :admin_user, parent: :user do
+    role 'admin'
+  end
+
   factory :restaurant do
```

Comment:
This factory would create the same restaurant over and over. It would probably make sense to sequence your name / address / etc.